src/containers/Guide/GuideBlocks/index.tsx

```diff
@@ -1,108 +1,118 @@
+          ],
+          ['Amy has enough coins in her account', ''],
+        ]}
+      />
       <p>
-        After Amy creates the block shown above, the network must validate several aspects of the block and all related
-        transactions including verification that:
+        See <NavLink to="/guide/transaction-fees">Transaction Fees</NavLink> for details on the fees that relate to this
+        transaction.
       </p>
-
-      <DocList variant="ol">
-        <li>Amy signed the block.</li>
-        <ol type="a">
-          <li>
-            The term "signing" is preferred over "created" because Amy may sign (authorize) blocks that she did not
-            originate.
-          </li>
-          <ol type="i">
-            <li>For example, a shopping website that builds the block as Amy adds items to her cart</li>
-          </ol>
-        </ol>
-        <li>Amy has enough coins in her account.</li>
-        <li>The block is not being validated more than once.</li>
-        <ol type="a">
-          <li>
-            This is critical for preventing banks and validators from sending or validating the same block multiple
-            times in the attempt to collect additional transaction fees
-          </li>
-        </ol>
-      </DocList>
-
+      <DocSubHeader>Dissecting a block</DocSubHeader>
       <p>
         The signing process produces digital signatures using the{' '}
-        <A href="https://ed25519.cr.yp.to/">Ed25519 Digital Signature Algorithm</A> to ensure that the account owner
-        indeed signed the set of transactions within a given block. Although in many explanations of the network the
-        account "owner" is often referred to by name, in the actual network architecture the individuals' names are
-        never stored. Instead, each account will refer to the owner by their account number.
+        <A href="https://ed25519.cr.yp.to/">Ed25519 Digital Signature Algorithm</A>. This ensures that the account owner
+        has indeed signed all the transactions within a block.
       </p>
+      <DocCallout type={CalloutType.note}>
+        Although in many explanations of the network the account "owner" is often referred to by name, the actual
+        network architecture never stores the name of individuals. Instead, each account refers to the owner by their
+        account number.
+      </DocCallout>
       <p>
-        The account number (often referred to as the "public-key" in public-key cryptography) is not only used to
-        identify your account when other users wish to send you coins, but it is also used during the verification
-        process in which blocks must be correctly verified to ensure that the sender (account owner) has authorized the
-        related transactions. Therefore, a more realistic representation of a block would be as follows.
+        The account number (in public-key cryptography often referred to as the "public-key") is not only important for
```

Comment:
The account number (often referred to as the "public-key" in cryptography)?

Reply:
Yeap, you are right